### ISSUE TITLE
No output for leaf nodes of the taxonomy

### ISSUE THREAD

**Issue by ialbert** [shenwei356/taxonkit#4]
Hello,

I am playing around with taxonkit and I got puzzled by its behavior. It looks like it will not produce output when searching for leaf nodes of the taxonomic tree. For example, this prints nothing:

`taxonkit list --show-rank --ids 9913`

same with lineage etc. Any time I search for a node with no children I get nothing back.

I am not sure I understand what is the rationale for this. Is this working as intended? 

How would one use `taxonkit` to figure out that `9913` is the the taxid  for `Bos Taurus`?
le

**Comment by shenwei356** [comment]
fixed in [v0.2.1](https://github.com/shenwei356/taxonkit/releases/tag/v0.2.1):

```
$ taxonkit list --show-rank --show-name --ids 9913
[INFO] parsing names file: /home/shenwei/.taxonkit/names.dmp
[INFO] 1593209 names parsed
[INFO] parsing nodes file: /home/shenwei/.taxonkit/nodes.dmp
[INFO] 1593209 nodes parsed
9913 [species] Bos taurus
```

**Comment by ialbert** [comment]
great, thanks!

**Comment by ialbert** [comment]
One more suggestion from a usability perspective. Having the tool always print debug information (sometimes even multiple times during one operation) is distracting from the task that one tries to accomplish. Sometimes it is hard to see the actual result because of this.

```
[INFO] parsing names file: /Users/ialbert/.taxonkit/names.dmp
[INFO] 1629550 names parsed
[INFO] parsing nodes file: /Users/ialbert/.taxonkit/nodes.dmp
[INFO] 1629550 nodes parsed
```

**Comment by shenwei356** [comment]
ok, i'll make them as optional output tomorrow.

**Comment by shenwei356** [comment]
[v0.2.2](https://github.com/shenwei356/taxonkit/releases/tag/v0.2.2)

```
$ taxonkit list --show-rank --show-name --ids 9913
9913 [species] Bos taurus

```

**Comment by ialbert** [comment]
nice thanks!

**Comment by shenwei356** [comment]
I'm so sorry, I just fixed a bug brought in v0.2.1 :( , see [v0.2.3](https://github.com/shenwei356/taxonkit/releases/tag/v0.2.2).
